includes/manage_collection.inc

```diff
@@ -328,7 +333,16 @@ function islandora_basic_collection_policy_management_form_submit(array $form, a
     $content_model = islandora_object_load($pid);
     $policy->addContentModel($pid, $properties['prompt'], $properties['namespace']);
   }
-  $collection['COLLECTION_POLICY']->content = $policy->getXML();
+  if (!isset($collection['COLLECTION_POLICY'])) {
+    $cp_ds = $collection->constructDatastream('COLLECTION_POLICY', 'M');
```

Comment:
They're usually X, not M, when created by solution packs. Meant I couldn't verify in Fedora Admin that the content was what I expected it to be. Other than that, great!